src/Nox.Generator/Application/DomainEventHandlers/DomainEventHandler.template.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Generated
+
+#nullable enable
+
+using MediatR;
+using Nox.Application;
+using Nox.Messaging;
+
+using {{codeGeneratorState.DomainNameSpace}};
+using {{codeGeneratorState.ApplicationNameSpace}}.Dto;
+
+namespace {{codeGeneratorState.ApplicationNameSpace}}.DomainEventHandlers;
+
+internal abstract class {{className}}Base : INotificationHandler<{{entity.Name}}{{crudOperation}}>
+{
+    private readonly IOutboxRepository _outboxRepository;
+
+    protected {{className}}Base(IOutboxRepository outboxRepository)
+    {
+        _outboxRepository = outboxRepository;
+    }
+
+    public virtual async Task Handle({{entity.Name}}{{crudOperation}} domainEvent, CancellationToken cancellationToken)
+    {
+{{- if raiseIntegrationEvent }}
+        var dto = domainEvent.{{entity.Name}}.ToDto();
+        var @event = new IntegrationEvents.{{entity.Name}}{{crudOperation}}(dto);
```

Comment:
for avoid conflicts add the full namespace
```suggestion
        var @event = new {{codeGeneratorState.??}}.IntegrationEvents.{{entity.Name}}{{crudOperation}}(dto);
```

Reply:
Changes were applied.